flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -414,7 +424,8 @@ private static void checkAndCreateDirectory(File directory) throws IOException {
 				nativeMetricOptions,
 				metricGroup,
 				restoreStateHandles,
-				ttlCompactFiltersManager);
+				ttlCompactFiltersManager,
+				writeBatchSize);
```

Comment:
Can we use `this.writeBatchSize` directly?

Reply:
ditto